Specs/Renderer/ShaderCacheSpec.js

```diff
@@ -112,7 +112,7 @@ defineSuite([
 
         var cache = new ShaderCache(context);
         var sp = cache.replaceShaderProgram({
-            shaderProgram : sp,
+            shaderProgram : sp, //eslint-disable-line no-use-before-define
```

Comment:
This is a bit shady.  What is the intension of this test?  Perhaps just removing `shaderProgram : sp` is OK?